src/media/js/apps_buttons.js

```diff
@@ -135,6 +135,12 @@ define('apps_buttons',
                 def.reject();
             });
         } else {
+            // Close the window if it was a paid app we'd logged-in for
```

Comment:
This is a case where it's a paid app and you're starting not logged in. This means we create the poup window for login and expect to keep around for a payment. However this case we've already got the app so the keeping the window around for payments is no longer needed. I'll fix up th e comment.